src/ML/src/MANNTrajectoryGenerator.cpp

```diff
@@ -361,6 +371,15 @@ bool MANNTrajectoryGenerator::advance()
         }
     }
 
+
+/*     for (const auto[key, list] : m_pimpl->contactListMap)
+    {
+        std::cerr << key << std::endl;
+
+        for(auto contact : list)
+            std::cerr << "position " << contact.pose.translation().transpose() <<  " activation " << std::chrono::duration<double>(contact.activationTime).count() << " deactivation " << std::chrono::duration<double>(contact.deactivationTime).count() << std::endl;
+    }
+ */
```

Comment:
Commented code

Reply:
Ops, removed in 5d9cc83df41de42783b30f656afa13314b0b58f3. Thank you 😄